qiskit/extensions/quantum_initializer/ucry.py

```diff
@@ -86,7 +86,17 @@ def ucy(self, angle_list, q_controls, q_target):
     if num_contr != len(q_controls):
         raise QiskitError("Number of controlled rotations does not correspond to"
                           " the number of control-qubits.")
-    return self.append(UCY(angle_list), [q_target] + q_controls, [])
+    return self.append(UCRYGate(angle_list), [q_target] + q_controls, [])
+
+
+def ucy(self, angle_list, q_controls, q_target):
+    """
+    Deprecated version of ucry.
+    """
+    import warnings
+    warnings.warn('qc.ucy is deprecated, use qc.ucry instead!', DeprecationWarning, 2)
```

Comment:
This message does not match the style of other deprecation messages.
```suggestion
    warnings.warn('The QuantumCircuit. ucy() method is deprecated as of 0.14.0, and '
                  'will be removed no earlier than 3 months after that release date. '
                  'You should use the QuantumCircuit. ucry() method instead.',
                  DeprecationWarning, stacklevel=2)
```